src/index.ts

```diff
@@ -1,123 +1,40 @@
 import * as puppeteer from 'puppeteer';
-import * as inquirer from 'inquirer';
-import * as Ajv from 'ajv';
-import * as moment from 'moment';
-import { fetchPoalimXSRFWithinPage, fetchGetWithinPage } from './utils/fetch';
-import accountDataSchemaFile from './schemas/accountDataSchema.json';
-import ILSCheckingTransactionsDataSchemaFile from './schemas/ILSCheckingTransactionsDataSchema.json';
-import foreignTransactionsSchema from './schemas/foreignTransactionsSchema.json';
-import { AccountDataSchema } from '../generatedTypes/accountDataSchema';
-import { ILSCheckingTransactionsDataSchema } from '../generatedTypes/ILSCheckingTransactionsDataSchema';
-import { ForeignTransactionsSchema } from '../generatedTypes/foreignTransactionsSchema';
-import { IncomingHttpHeaders } from 'http';
-import { v4 as uuidv4 } from 'uuid';
-import * as fetch from 'node-fetch';
 import { config } from 'dotenv';
+import * as scraper from './scrapers/scrapersIndex';
+import * as browserUtil from './utils/browserUtil';
 config();
 
 declare global {
   namespace NodeJS {
     interface ProcessEnv {
-      USER_CODE: string;
-      PASSWORD: string;
+      USER_CODE: string; // suggested: rename 'HAPOALIM_BUISNESS_USER_CODE'
+      PASSWORD: string; // suggested: rename 'HAPOALIM_BUISNESS_PASSWORD'
```

Comment:
yes let's do that

Reply:
If we're using business flag, than we could use the same variables.
Do you think there could be a case of user with both personal and bisiness accounts?

Reply:
maybe but if we don't have it now, no need to take care of it...

Reply:
removed